### PR TITLE
Loading state for tx table front page

### LINKED ISSUE #1132: loading state for tx table front page

### PR DESCRIPTION
**Pull request by prin-r**
Fix #1132

### CODE REVIEW

**Comment by prin-r** [comment]
This PR is waiting for #1025 (Implement loading animation for all cards). However everyone can review anyway.

File: scan/src/components/LatestTxTable.re [hunk 1]
```diff
@@ -34,136 +34,173 @@ module Styles = {
...
+  </TBody>;
+};
+
+let rec createNoDataList = n =>
+  if (n <= 0) {
+    [];
+  } else {
+    [ApolloHooks.Subscription.NoData, ...createNoDataList(n - 1)];
+  };
+
```
Comment (evilpeach) [inline scan/src/components/LatestTxTable.re]: Let's use `Belt.List.make` or `Belt.Array.make`

File: scan/src/components/LatestTxTable.re [hunk 2]
```diff
@@ -34,136 +34,173 @@ module Styles = {
...
+            />
+          </div>
+        </Col>
+        <HSpacing size={`px(20)} />
+      </Row>
+    </THead>
+    {switch (allSub) {
+     | Data((txs, _)) =>
+       txs
+       ->Belt_Array.mapWithIndex((i, e) => txBodyRender(i, ApolloHooks.Subscription.Data(e)))
```
Comment (evilpeach) [inline scan/src/components/LatestTxTable.re]: Don't use `ApolloHooks.Subscription.Data`. Let's use `Sub.resolve`.

File: scan/src/components/LatestTxTable.re [hunk 3]
```diff
@@ -34,136 +34,173 @@ module Styles = {
...
+       ->React.array
+     | _ =>
+       createNoDataList(10)
```
Comment (evilpeach) [inline scan/src/components/LatestTxTable.re]: Use `Belt.Array.make`.

File: scan/src/components/LatestTxTable.re [hunk 4]
```diff
@@ -34,136 +34,173 @@ module Styles = {
...
+        size=Text.Xxl
+        weight=Text.Bold
+        block=true
+        color=Colors.gray8
+      />
+      <div className=Styles.seeAll onClick={_ => Route.redirect(Route.TxHomePage)}>
+        <div className=Styles.cFlex>
+          {switch (allSub) {
+           | Data((_, totalCount)) =>
+             <span className=Styles.amount> {totalCount |> Format.iPretty |> React.string} </span>
```
Comment (evilpeach) [inline scan/src/components/LatestTxTable.re]: Let's remove the amount style and use `Text`.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — CHANGES_REQUESTED** [review]
image
The msg in the action column exceeds the table.

**Review by evilpeach — APPROVED** [review]

**Review by tansawit — APPROVED** [review]

**Pull request merged**